components/form/rangeDatepicker/src/index.js

```diff
@@ -122,7 +123,15 @@ FormRangeDatepicker.propTypes = {
   /**
    * Init date of the selected range
    */
-  startDate: PropTypes.instanceOf(Date)
+  startDate: PropTypes.instanceOf(Date),
+  /**
+   * Icon of select inputs
+   */
+  icon: PropTypes.func
```

Comment:
Shouldn't it be "node" instead of func?

Reply:
I used the same method used in: https://github.com/SUI-Components/sui-components/blob/master/components/tag/selectable/src/index.js#L42